src/RTCUtil.js

```diff
@@ -112,6 +112,14 @@ function normalizeMediaConstraints(constraints, mediaType) {
     }
 }
 
+function chr4() {
+    return Math.random().toString(16).slice(-4);
+}
+
+export function uniqueID() {
```

Comment:
Can you please add some docs here explaining why we are doing it this way?

Reply:
Hopefully good enough 👍🏻